.npmignore

```diff
@@ -0,0 +1,12 @@
+## Directories
+examples/
+build/
+test/
+doc/
+
+## Files
```

Comment:
I believe CONTRIBUTING.md and README.md must stay. Also you might want to fiddle the source often, so I won't leave out webpack config files.